src/content/faq/General.md

```diff
@@ -8,20 +8,82 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI components](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
+* Create a representation of your data that feels familiar and natural ([a graph](/learn/thinking-in-graphs/#it-s-graphs-all-the-way-down-https-en-wikipedia-org-wiki-turtles-all-the-way-down)).
+* Ensure that all of your data is [statically typed](/learn/schema/) and these types inform [what queries the schema supports](/learn/introspection/).
+* [Reduce the need for breaking changes](/learn/best-practices/#versioning), but utilize a [built-in mechanism for deprecations](https://spec.graphql.org/draft/#sec-Deprecation) when you need to.
+* Examine queries and responses through community tools like [GraphiQL](https://github.com/graphql/graphiql).
+
+And many others! [Our homepage](/) outlines some more convincing reasons to use GraphQL.
+
+Additionally, it doesn’t take rewriting your entire application to try out GraphQL. You can even start with a single HTTP request that wraps an existing REST call, for instance. Your [GraphQL schema](/learn/thinking-in-graphs/#shared-language) and [business domain model](/learn/thinking-in-graphs/#business-logic-layer) can be expanded gradually, so we’d recommend focusing on one use case at first and only build the part of the schema you need for that to work. 
```

Comment:
I've condensed this a little; feel free to dismiss.

```suggestion
You can try out GraphQL without rewriting your entire application; for example: start with a single HTTP request that wraps an existing REST call. Your [GraphQL schema](/learn/thinking-in-graphs/#shared-language) and [business domain model](/learn/thinking-in-graphs/#business-logic-layer) can be expanded gradually; we recommend focusing on one use case at first and only building the part of the schema needed for that. 
```

Reply:
Definitely agree with condensing but confession: I hate semicolons 😂  So I'll apply the changes - just without those 😁